.claude/scripts/rebuild-diagrams.sh

```
@@ -0,0 +1,91 @@
+    [ ! -f "$svg" ] && return 0
+    [ "$src" -nt "$png" ] && return 0
+    [ "$src" -nt "$svg" ] && return 0
+    return 1
+}
+
+rebuilt=0
+skipped=0
+failed=0
+cd "$REPO_ROOT/.claude/skills/excalidraw-diagram/references"
+for src in "${TARGETS[@]}"; do
+    rel="${src#$REPO_ROOT/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)